### ISSUE TITLE
it's blocking playback

### ISSUE THREAD

**Issue by rinormaloku** [igrigorik/videospeed#592]
Currently, it's blocking playback on youtube. It used to work perfectly there must have been some updates from YT side to stop this tool.

**Comment by ChadBailey** [comment]
I'm not sure what you're referring to. VSC continues to work correctly for me. I do have YT Premium, so I even tried logging out to try it with ads and it still continues working correctly.

Can you give steps to reproduce?

**Comment by rinormaloku** [comment]
Sadly no as I am not aware of what triggered it. At one point youtube stopped playing videos (after it being in autoplay for hours).

Refreshing the video again played for ca 4 seconds and blocked again. I went on blocking all extensions one by one to verify which is causing the problem. As such, I got to the conclusion that this extension is blocking it. 
Enabled the extension today again, blocked went into the developer tools of it and it started working xD

So I am clueless but at least this is resolved on my side and the issue can be closed.
